modules/mono/mono_gd/gd_mono_android.cpp

```diff
@@ -494,11 +489,9 @@ static void interop_get_active_network_dns_servers(char **r_dns_servers, int *dn
 	// The JNI code is the equivalent of:
 	//
 	// ConnectivityManager connectivityManager = (ConnectivityManager)getApplicationContext()
-	// 		.getSystemService(Context.CONNECTIVITY_SERVICE);
-	// Network activeNerwork = connectivityManager.getActiveNetwork();
-	// LinkProperties linkProperties = connectivityManager.getLinkProperties(activeNerwork);
+
 	// List<String> dnsServers = linkProperties.getDnsServers().stream()
-	// 		.map(inetAddress -> inetAddress.getHostAddress()).collect(Collectors.toList());
+	//  	.map(inetAddress -> inetAddress.getHostAddress()).collect(Collectors.toList());
```

Comment:
The commented code in this file should not be removed. It's the reference java code for the unreadable JNI boilerplate.